src/__tests__/vdxf/appencryptionrequestdetails.test.ts

```diff
@@ -0,0 +1,80 @@
+import { BN } from "bn.js";
+import { AppEncryptionRequestDetails, AppEncryptionRequestDetailsJson, CompactIdAddressObject } from "../../vdxf/classes";
+import { TransferDestination } from "../../pbaas/TransferDestination";
+import { BigNumber } from "../../utils/types/BigNumber";
+
+// Helper function to create TransferDestination from address string
+function createCompactIdAddressObject(flags: BigNumber, address: string): CompactIdAddressObject {
+  const obj = new CompactIdAddressObject();
+  obj.flags = flags;
+  obj.address = address;
+  return obj;
+}
+
+
+describe("AppEncryptionRequestDetails serialization tests", () => {
+  test("creates valid AppEncryptionRequestDetails with zaddress", () => {
+    const details = new AppEncryptionRequestDetails();
```

Comment:
Use the constructor here to make sure it works as intended instead of setting class properties manually